libkbfs/choosedefaultmount_windows.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+// +build windows
+
+package libkbfs
+
+import (
+	"errors"
+	"strings"
+)
+
+func chooseDefaultMount(dirs []string) (string, error) {
+	var dir string
+	if len(dirs) == 0 {
+		return "K:", errors.New("chooseDefaultMount fails - nothing to choose from")
```

Comment:
I intended for this to fall back to what we do now, in case the detection/availability logic failed. K: may accidentally work, where "" definitely will not - I don't see the disadvantage of using K: here. But I'll change it.